OxfordBodleian/Juel-Jensen/BDLaethg40.xml

```diff
@@ -37,9 +37,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="1r1" to="1r121"/>
+                            <title type="complete" ref="LIT1763Legend"/>
+                            <textLang mainLang="gez"/>
+                            <note>The legend of Susǝnyos proper is framed by lengthy protective
+                            supplications.</note>
+                            <incipit xml:lang="gez" type="inscriptio"><hi rend="rubric">በስመ፡ እግዚአብሔር፡ ሕያው፡ ነባቢ፡ ወተናጋሪ፡ ጸሎቱ፡ ለቅዱስ፡ ሱስንዮስ፡ በ</hi>እንተ፡
+                            አሰስሎ፡ ደዌ፡ እምሕፃናት፡ እለ፡ ይጠብው፡ ጥበ<supplied reason="omitted">፡</supplied> እሞሙ፡ እምኵሉ፡ 
+                                ደዊያ<hi rend="rubric"><gap reason="illegible" unit="chars" quantity="1"/>፡ ዓዲ፡ ጸሐፍኩ፡ ለብእሲ<surplus>ሲ</surplus>ት፡ ሐሐዋ፡ ዘየሐዩ፡ 
+                                ደዌ</hi><gap reason="illegible"/> ወትስቅሎ፡ ላዕሌሃ፡ ወይበቁዕ፡ በረድኤተ፡ እግዚአብሔር፡ ልዑል፡ ወክቡር፡
```

Comment:
I would mark ወይበቁዕ፡ with `<sic/>`, the expected form should be ወይበቍዕ፡ (the scroll has indeed ወይበቁዕ)